jboss/container/prometheus/7/backwards_compatibility.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -ueo pipefail
+
+# Legacy location
+ln -s /usr/share/java/prometheus-jmx-exporter/jmx_prometheus_javaagent.jar \
```

Comment:
if the source location don't exist this this script will fail and the image build fail, right?
maybe add a check to make sure that this will not happen?